tools/autodeployment/deploy.sh

```diff
@@ -34,7 +34,7 @@ fi
 echo "GIT_COMMIT=$2" >> $ENV_FILE
 
 echo "Building $ENV Container"
-docker-compose --env-file $ENV_FILE --project-name=$ENV build
+GITHUB_TOKEN={$GITHUB_TOKEN} docker-compose --env-file $ENV_FILE --project-name=$ENV build
```

Comment:
is there anyway we can see the value of `GITHUB_TOKEN` here?

Reply:
It would have to be set outside the script.  @manekenpix can speak to what's possible in the env of the build machines.

Reply:
I think it should be set [in here](https://github.com/Seneca-CDOT/telescope/blob/master/tools/autodeployment/env.example) so the `autodeployment` server can see it, and then passed into the script.

Reply:
Oh, you already did, the only step left is passing it to the script [here](https://github.com/Seneca-CDOT/telescope/blob/184a18dcc5ba9d25e67c48e957114e08be071353/tools/autodeployment/builds.js#L81)

Reply:
Not sure if it belongs to this PR, but we could also document this change [in our docs](https://github.com/Seneca-CDOT/telescope/blob/master/docs/staging-production-deployment.md#3--autodeployment-server)